checkpoint_harmony_endpoint/README.md

```diff
@@ -0,0 +1,77 @@
+# Agent Check: checkpoint_harmony_endpoint
+
+## Overview
+
+Checkpoint Harmony Endpoint is a next-generation endpoint security solution designed to prevent, detect, and respond to threats on user devices (desktops, laptops, servers, etc). This integration monitors [Checkpoint Harmony Endpoint][1].
```

Comment:
```suggestion
Checkpoint Harmony Endpoint is a next-generation endpoint security solution designed to prevent, detect, and respond to threats on user devices, such as desktops, laptops, and servers. This integration monitors [Checkpoint Harmony Endpoint][1].
```